### ISSUE TITLE
CLI Warnings

### ISSUE THREAD

**Issue by Eomm** [MattiLehtinen/postgrator-cli#27]
Hi

Running `postgrator` (cli) with the example `postgrator.json`

```
{
  "migrationDirectory": "migrations",
  "driver": "pg",
  "host": "127.0.0.1",
  "port": 5433,
  "database": "postgres",
  "username": "postgres",
  "password": "postgres"
}
```

Lead to  this output

```
postgrator deprecated Config option "port". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:222:26
postgrator deprecated Config option "host". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:222:26
postgrator deprecated Config option "username". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:222:26
postgrator deprecated Config option "password". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:222:26
postgrator deprecated Config option "migrationDirectory". use "migrationPattern" instead using glob match. e.g. path.join(__dirname, '/migrations/*') node_modules/postgrator-cli/postgrator-cli.js:222:26
```

The `postgrator` version is 4.3.1

I saw that these warnings have been introduced in 4.3.0
https://github.com/rickbergfalk/postgrator/pull/131

**Comment by perrin4869** [comment]
This is fixed in my fork [perrin4869/postgrator-cli](https://www.npmjs.com/package/@perrin4869/postgrator-cli), hoping to have it merged via this [PR](https://github.com/MattiLehtinen/postgrator-cli/pull/30)

**Comment by Eomm** [comment]
I hope @MattiLehtinen get the change to review it!

**Comment by perrin4869** [comment]
Fixed in #30
